lib/platform-stats.js

```diff
@@ -15,15 +15,27 @@ export const updatePlatformStats = async (pgClient, honestMeasurements) => {
  * @param {import('./preprocess').Measurement[]} honestMeasurements
  */
 export const updateDailyStationStats = async (pgClient, honestMeasurements) => {
-  // TODO: when we add more fields, we will update the ON CONFLICT clause
-  // to update those fields, and we won't just use a Set for the stationIds
-  // which currently removes all granular measurement details
-  const stationIds = [...new Set(honestMeasurements.map(m => m.stationId))]
-  debug('Updating daily station stats, unique_count=%s', stationIds.length)
+  const measurementCountsPerStation = new Map()
+  for (const m of honestMeasurements) {
+    measurementCountsPerStation.set(
+      m.stationId,
+      (measurementCountsPerStation.get(m.stationId) || 0) + 1
+    )
+  }
 
+  debug('Updating daily station stats, unique_count=%s', measurementCountsPerStation.size)
+
+  // Coalesce to 0 to avoid null + count = null
+  // Use EXCLUDED to allow incrementing the count
```

Comment:
Do we still need to use EXCLUDED now that `accepted_measurement_count` is never NULL?

Reply:
I believe so. EXCLUDED is for the fact that there was a conflict. For example, see public-stats.js `mapParticipantsToIds`

I did remove COALESCE because it's never NULL now, but I forgot to remove the comment.